scripts/release.js

```diff
@@ -111,14 +110,41 @@ fs.writeFileSync(
 spawnOrFail('npm', [`version ${versionString} --no-git-tag-version`]);
 logger.log(`Updated package.json version to ${versionString}`);
 
+// Update the peer dependency to the most updated version of the SDK
+const updatedSdkVersion = spawnOrFail('npm', [`show amazon-chime-sdk-js version`]).trim();
+
+logger.log(`Installing SDK Version: ${updatedSdkVersion} into the meeting demo, chat demo, and as a peerDependency and devDependency of the react library.`);
+
+let componentsPackageJson = JSON.parse(fs.readFileSync('package.json', 'utf-8'));
+const oldSdkVersion = componentsPackageJson.peerDependencies['amazon-chime-sdk-js'];
+let replacedPeerDependency = fs.readFileSync('package.json').toString();
+// Replace first instance
+replacedPeerDependency = replacedPeerDependency.replace(`"amazon-chime-sdk-js": "${oldSdkVersion}"` , `"amazon-chime-sdk-js": "^${updatedSdkVersion}"`);
```

Comment:
Can we pass this out to JSON and set devDependency and peerDependency correctly? :D

Reply:
Done, thanks